common/analysis/processingChainValidator.js

```diff
@@ -31,24 +32,59 @@ export const validateChain = async (chain, defaultLang) =>
     ],
   })
 
+const _validateStepVariablesPrevStep = (step) => {
+  const validation = Validation.newInstance()
+  const variablesPreviousStep = ProcessingStep.getVariablesPreviousStep(step)
+  variablesPreviousStep.forEach((variable, index) => {
+    if (
+      ProcessingStepVariable.getInclude(variable) &&
+      StringUtils.isBlank(ProcessingStepVariable.getAggregate(variable))
+    ) {
+      R.pipe(
```

Comment:
does this modify the passed validation?

Reply:
it modifies the validation object created at line 36